yb-voyager/cmd/exportData_test.go

```diff
@@ -659,3 +749,330 @@ func testCasesWithDifferentTableListFlagValuesTest1(t *testing.T, firstRunTableL
 	assertInitialTableListOnSubsequentRun(t, false, firstRunTableList, firstRunPartitionsToRootMap)
 
 }
+
+func TestTableListInSubsequentRunOfExportDatWithTableListFlagsPGTest2(t *testing.T) {
```

Comment:
Would recommend being more descriptive in the test name. `2` is not very helpful. It seems like we're testing "partitions" here?

Reply:
testing multi-level partiitons and glob pattern cases in table-list flags